components/Shipping/models.rst

```diff
@@ -0,0 +1,186 @@
+| category            | e.g. "Regular"                                                          |
++---------------------+-------------------------------------------------------------------------+
+| categoryRequirement | Reference to constant from ShippingMethodInterface                      |
++---------------------+-------------------------------------------------------------------------+
+| enabled             | Boolean flag of enablement                                              |
++---------------------+-------------------------------------------------------------------------+
+| calculator          | Reference to constant from DefaultCalculators                           |
++---------------------+-------------------------------------------------------------------------+
+| configuration       | Extra configuration for calculator                                      |
++---------------------+-------------------------------------------------------------------------+
+| rules               | Collection of Rules                                                     |
++---------------------+-------------------------------------------------------------------------+
+| createdAt           | Creation time                                                           |
++---------------------+-------------------------------------------------------------------------+
+| updatedAt           | Last update time                                                        |
++---------------------+-------------------------------------------------------------------------+
+| currentTranslation  | Translation chosen from translations list accordingly to current locale |
++---------------------+-------------------------------------------------------------------------+
+| currentLocale       | Currently set locale                                                    |
++---------------------+-------------------------------------------------------------------------+
+| translations        | Collection of translations                                              |
++---------------------+-------------------------------------------------------------------------+
+| fallbackLocale      | Locale used in case no translation is available                         |
++---------------------+-------------------------------------------------------------------------+
+
+.. note::
+    This model implements the :ref:`component_shipping_model_shipping-method-interface` and extends
+    :ref:`component_translation_model_abstract-translatable` class. |br|
+    For more detailed information go to `Sylius API ShippingMethod`_.
+
+.. _Sylius API ShippingMethod: http://api.sylius.org/Sylius/Component/Shipping/Model/ShippingMethod.html
+
+ShippingMethodTranslation
+-------------------------
+
+**ShippingMethodTranslation** object allows translate the name of shipping method according to the given locales.
```

Comment:
`allows to translate the shipping method's name accordingly to the provided locales.`